tests/spread/integration/apache2/task.yaml

```diff
@@ -0,0 +1,30 @@
+summary: Integration tests for apache2
+
+execute: |
+  rootfs="$(apache2_bins dash_bins base-files_bin coreutils_bins adduser_bins curl_bins)"
```

Comment:
A couple comments:
- Why do we need `base-files_bin` & `coreutils_bins`?
- I think we can omit `curl_bins` if we don't run `curl` in the chroot?